infinite_tracing/lib/infinite_tracing/client.rb

```diff
@@ -143,8 +147,10 @@ def stop
 
       def start_streaming(exponential_backoff = true)
         return if suspended?
+
+        @exponential_backoff = exponential_backoff
```

Comment:
How come we're saving this here?

Reply:
Whoops! This was leftover from an earlier version of the code. I was using an instance var instead of simply baton passing this method's lone argument further down to the next method, but passing the method ended up working best. Addressed with 873ed0a48aabb6c4137d69c2bc5a20457733553c